packages/types/src/browser-proxy/index.ts

```diff
@@ -31,6 +31,8 @@ export interface IBrowserProxyPlugin {
 
     url(applicant: string, val: string): Promise<any>;
 
+    newWindow(applicant: string, val: string, windowName: string, windowFeatures: object): Promise<any>;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/open#Window_features - here is a window features list to define in object.